test/fork/assets/USDC.t.sol

```diff
@@ -3,25 +3,26 @@ pragma solidity >=0.8.22 <0.9.0;
 
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
-import { CreateWithTimestamps_LockupDynamic_Batch_Fork_Test } from "../batch/createWithTimestampsLD.t.sol";
-import { CreateWithTimestamps_LockupLinear_Batch_Fork_Test } from "../batch/createWithTimestampsLL.t.sol";
-import { CreateWithTimestamps_LockupTranched_Batch_Fork_Test } from "../batch/createWithTimestampsLT.t.sol";
+import { CreateWithTimestamps_LockupDynamic_Batch_Lockup_Fork_Test } from "../batch-lockup/createWithTimestampsLD.t.sol";
+import { CreateWithTimestamps_LockupLinear_Batch_Lockup_Fork_Test } from "../batch-lockup/createWithTimestampsLL.t.sol";
+import { CreateWithTimestamps_LockupTranched_Batch_Lockup_Fork_Test } from
```

Comment:
`BatchLockup` is the name of a single contract so let's remove the underscore between these words.